test/gtest/uct/ib/test_dc.cc

```diff
@@ -448,10 +448,13 @@ UCS_TEST_P(test_dc_flow_control, soft_request)
     EXPECT_EQ(get_fc_ptr(m_e1)->fc_wnd, s_thresh - 1);
 }
 
-/* Check that flush returns UCS_OK even if there is an outgoing grant request */
+/* Check that:
+ * 1) flush returns UCS_OK even if there is an outgoing grant request
```

Comment:
how is this tested?

Reply:
we send the whole window of messages which means that grant request was sent as well.
Then we perform a flush and check its return status. Need to do that in a loop, because we may not poll all send completions before the first poll call.